src/main/java/space/npstr/magma/connections/hax/ClosingUndertowWebSocketClient.java

```diff
@@ -111,6 +100,12 @@ private void handleChannelPatched(final URI url, final WebSocketHandler handler,
         handler.handle(session).subscribe(session);
     }
 
+    private HandshakeInfo createHandshakeInfo(final URI url, final DefaultNegotiation negotiation) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Rename this method; there is a "private" method in the parent class with the same name. ([squid:S2177](https://sonarcloud.io/organizations/napstr-github/rules#rule_key=squid%3AS2177))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=space.npstr.magma%3Amagma&pullRequest=14&issues=AWas-MiDFsWw174ZRTe8&open=AWas-MiDFsWw174ZRTe8)

[ISSUE_KEY]: AWas-MiDFsWw174ZRTe8